content/aips/AAVE_ETHEREUM_V3.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Aave Ethereum V3
+author: BGD Labs (@bgdlabs)
+shortDescription: Listing WBTC, WETH, wstETH, USDC, DAI, LINK, AAVE and activating Ethereum V3 Pool
+discussions: https://governance.aave.com/t/bgd-aave-v3-ethereum-new-deployment-vs-aave-v2-upgrade/9990/16
+created: 2023-01-20
+---
+
+# Simple Summary
+
+This governance proposal executes the listing og WBTC, WETH, wstETH, USDC, DAI, LINK, AAVE on AAVE Ethereum V3.0.1 pool. It also creates an `ETH correlated` eMode category and activates the pool.
+
+# Motivation
+
+While AAVE V3 was deployed to various networks right after the release, the Ethereum pool was still running V2.
+After careful considerations, the community decided to deploy a fresh V3 instead of upgrading the V2 pool for increased compatibility between the V3 pools.
```

Comment:
```suggestion
After careful consideration, the community [decided to deploy a fresh V3](https://snapshot.org/#/aave.eth/proposal/0x584eb4e0f79e1d9dcdd99b3a0c831bfc3c654af3f8f619d5f68eae23cd9cb149) instead of upgrading the V2 pool, for increased compatibility between the V3 pools and less general complexity.
```